src/encoded/vis_defines.py

```diff
@@ -451,8 +453,9 @@ class VisDefines(object):
     # Loads vis_def static files and other defines for vis formatting
     # This class is also a swiss army knife of vis formatting conversions
 
-    def __init__(self, dataset=None):
+    def __init__(self, request, dataset=None):
         # Make these global so that the same files are not continually reloaded
+        self.request = request
```

Comment:
same here, 

request is an internal var to the class so it should start with an underscore.